core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -8077,17 +8076,24 @@ components:
             An optional field which is only provided if the `radix_engine_toolkit_receipt`
             flag is set to true when requesting a transaction preview from the API.
 
-            This receipt is primarily intended for use with the toolkit and may contain information 
-            that is already available in the receipt provided in the `receipt` field of this 
+            This receipt is primarily intended for use with the toolkit and may contain information
+            that is already available in the receipt provided in the `receipt` field of this
             response.
 
-            A typical client of this API is not expected to use this receipt. The primary clients 
-            this receipt is intended for is the Radix wallet or any client that needs to perform 
+            A typical client of this API is not expected to use this receipt. The primary clients
+            this receipt is intended for is the Radix wallet or any client that needs to perform
             execution summaries on their transactions.
         instruction_resource_changes:
```

Comment:
Personally I'd rather not introduce a flag if we're deprecating it, because the flag would have to default to `true` anyway for backwards compatibility, so we don't get any real use out of it, it will just be a pain because we'll have to have a deprecated flag for another protocol update after the feature is removed :)